user-guide/rule-book/rule-book.md

```diff
@@ -899,35 +899,19 @@ owned by Hasbro ®).
 
 ## Links
 
-TripleA Downloads:
-Place to Report Bugs:
+[TripleA Downloads](https://sourceforge.net/projects/triplea/files/)
+[Place to Report Bugs](http://sourceforge.net/tracker/?group_id=44492)
```

Comment:
Ah these links are terribly out of date. Would you mind changing them?
Our official web presence can be found at
https://triplea-game.org/
https://forums.triplea-game.org/

and https://github.com/triplea-game/triplea for all code related stuff.
On a second thought it might be better to just remove the links entirely, since people reading this will be on our website anyways. (This includes the links below)

Reply:
Sure thing, I can do that. You have a good point, the link to github is also available on the footer so should be unnecessary. I've removed the whole link section on this PR. In the issues, its also been mentioned that the credits will need to be struck off so I've removed that as well.